app/models/bus_stop.rb

```diff
@@ -245,6 +245,10 @@ def decide_if_completed_by(user)
     end
   end
 
+  def add_hastus_id_to_name
+    name + " (#{hastus_id})"
```

Comment:
Might as well string interpolate the whole thing.
`"#{name} (#{hastus_id})"`

Function should probably be renamed, add makes it sound like it makes a permanent change to name rather than just returning them combine. Something like `name_with_hastus_id` is more clear what it does.